### PR TITLE
Dark timeline entries for the dark Nextcloud theme

### PR DESCRIPTION
**Pull request by mashedkeyboard**
* Resolves: Issue not reported, just fixed
* Target version: master 

### Summary
The hover colour of timeline posts is always white at present, even with the dark theme enabled. This is very jarring for dark theme users. This PR contains an incredibly simple change to introduce a lighter dark hover highlight for dark theme users.

### CODE REVIEW

File: src/components/TimelineEntry.vue [hunk 1]
```diff
@@ -138,6 +138,9 @@ export default {
 	.timeline-entry:hover {
 		background-color: #F5F5F5;
```
Comment (jancborchardt) [inline src/components/TimelineEntry.vue:139]: If you simply use the variable `var(--color-background-hover)` here, they are both standardized and you don’t need the extra rule for the .dark class. :)

Comment (mashedkeyboard) [reply src/components/TimelineEntry.vue]: That would, however, require me to know what I was doing :wink: sorted, should be good now! Thanks :smile:

**Review by jancborchardt — APPROVED** [review]
Looks good now! :) 

Something we should also do – but that’s something for a future pull request – is to make sure everything also works with keyboard-only. So have :focus states next to :hover, and make sure all the things you can click also work with enter. :)

**Comment by jancborchardt** [comment]
Welcome to Nextcloud, @mashedkeyboard! We have a Nextcloud Talk instance with a channel for Social app development, where I could invite you with a guest account if you like? :)

**Comment by mashedkeyboard** [comment]
Thank you, @jancborchardt! I'd like that.

**Pull request merged**